app/Models/Post.php

```diff
@@ -72,4 +73,19 @@ public function reactions()
     {
         return $this->hasMany(Reaction::class);
     }
+
+    /**
+     * Get the URL for the media for this post.
+     */
+    public function getMediaUrl()
+    {
+        if ($this->url === 'default') {
+            return 'default';
+        }
+
+        // Ask the storage driver for the path to the image for this post.
+        $path = Storage::url('uploads/reportback-items/edited_' . $this->id . '.jpeg');
```

Comment:
Should we add a `@TODO` here to update this lookup when we don't have to rely on `editied_` being in the url? could be something we forget. But totally not needed! Whatever you think is best